### DISCUSSION TITLE
Remove people I don't sponsor from my feed.

### DISCUSSION THREAD

**Discussion by Buildstarted** [community/community#112876] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

A person I no longer follow or sponsor keeps showing up in my Home feed. I have not sponsored them for well over a month. (Last day when yearly renewal happened)

`You're seeing this because you sponsor <user>`

**Comment by overtoneblue** [comment]
Why is this still an issue? one time sponsorship from months and months ago still shows up in the home feed unless the user is blocked.
